triggers/ekwipunek.lua

```diff
@@ -396,3 +397,8 @@ function wyswietlZawartoscPlecaka(zawartosc)
 
     cecho(linia .. "\n")
 end
+        end
+    end
+
+    cecho(linia .. "\n")
+end
```

Comment:
To nie wyglada dobrze.. Nie skopiowalo sie przez przypadek?

Reply:
Poprawione.